skpro/distributions/burr_iii.py

```diff
@@ -0,0 +1,64 @@
+"""Burr III probability distribution for skpro."""
+
+from scipy.stats import burr12, rv_continuous
+
+from skpro.distributions.adapters.scipy import _ScipyAdapter
+
+
+class BurrIII(_ScipyAdapter):
+    r"""Burr III probability distribution.
+
+    The Burr III distribution is a continuous probability distribution with two
+    parameters: shape parameter $c > 0$ and scale parameter $s > 0$.
+    Its probability density function (PDF) is:
+
+    .. math::
+        f(x; c, s) = \frac{c}{s} \left(\frac{x}{s}\right)^{-c-1}
+        \left[1 + \left(\frac{x}{s}\right)^{-c}\right]^{-2}, \quad x > 0
+
+    Parameters
+    ----------
+    c : float
+        Shape parameter
+    scale : float
+        Scale parameter
+    """
+
+    _tags = {
+        "authors": ["your-github-id"],
```

Comment:
again, I feel you did not check the code properly. AI assistants will do things like this if you do not tell them, say, your GitHub ID.